### PR TITLE
add a separator between views

### PR DESCRIPTION
**Pull request by bernard-ng**

### CODE REVIEW

File: sass/_controlbox.scss [hunk 1]
```diff
@@ -303,6 +303,7 @@
 
         .controlbox-panes {
             background-color: var(--controlbox-pane-background-color);
+            border-right: 3px solid var(--chatroom-head-bg-color);
```
Comment (jcbrand) [inline sass/_controlbox.scss:306]: You can't put a border to the right of the controlbox pane. What if the user opens a single-user chat, which is green? Then you'll see an orange border for a green chat.

File: sass/_controlbox.scss [hunk 2]
```diff
@@ -303,6 +303,7 @@
 
         .controlbox-panes {
             background-color: var(--controlbox-pane-background-color);
```
Comment (jcbrand) [inline sass/_controlbox.scss:305]: Hi @bernard-ng Can you please provide a screenshot of how it looks now?

**Comment by bernard-ng** [comment]
Screenshot from 2020-07-13 17-23-35

Screenshot from 2020-07-13 17-23-49

Screenshot from 2020-07-13 17-27-31

**Comment by jcbrand** [comment]
Thanks @bernard-ng, looks nice. I think it could look nice if you make the border between the occupants list and the chat history the same.


87575375-1cfc1080-c6d0-11ea-9073-570476e073ea

**Comment by bernard-ng** [comment]
Screenshot from 2020-07-17 16-40-06

**Comment by jcbrand** [comment]
Thanks @bernard-ng, please just rename `controlbox-pane-border-color` to `panel-divider-color`, then I'll merge.

File: sass/_variables.scss [hunk 3]
```diff
@@ -154,7 +154,7 @@ $mobile_portrait_length: 480px !default;
 
     --occupants-padding: 1em;
     --occupants-background-color: white;
-    --occupants-border-left: 3px solid var(--controlbox-pane-border-color);
+    --occupants-border-left: 3px solid var(--panel-divider-color);
```
Comment (jcbrand) [inline sass/_variables.scss:157]: Hi @bernard-ng, sorry, could you please make one more change.

We should try avoiding absolute measurements such as pixels, and instead use relative ones like `em` or `rem`.

Can you please set `3px` to how many `rem` is necessary to have the same effect?

**Pull request merged**